interface/main/calendar/find_appt_popup.php

```diff
@@ -418,12 +402,12 @@ function setappt(year,mon,mday,hours,minutes) {
     $atitle = "Choose ".date("h:i a", $utime);
     $adate = getdate($utime);
     $anchor = "<a href='' onclick='return setappt(" .
-    $adate['year'] . "," .
-    $adate['mon'] . "," .
-    $adate['mday'] . "," .
-    $adate['hours'] . "," .
-    $adate['minutes'] . ")'".
-    " title='$atitle' alt='$atitle'".
+    attr_js($adate['year']) . "," .
```

Comment:
hi @attritionorg , since this is in js that is inside html, for xss, this function will do both json_encode() and htmlspecialchars() (xss)